superchain/superchain.go

```diff
@@ -278,7 +281,7 @@ func (c ContractVersions) Check() error {
 			return fmt.Errorf("invalid type for field %s", val.Type().Field(i).Name)
 		}
 		if str == "" {
-			return fmt.Errorf("empty version for field %s", val.Type().Field(i).Name)
+			continue // we allow empty strings and rely on tests to assert (or except) a nonempty version
```

Comment:
Makes sense. Though we'll have to be careful in the future as we version new superchain contracts. I wonder if this behavior can be configured by the caller? Say something like `Check(skipEmptyVersions)`.